lib/components/first-time-point/style.less

```diff
@@ -5,15 +5,17 @@
 
   @{prefix}arrow {
     left: -45px;
-    position: relative;
+    position: absolute;
+    top: -62px;
   }
 
   @{prefix}arrow-pointer {
-    width: 100px;
-    height: 60px;
+    width: 88px;
+    height: 37px;
     position: relative;
+    left: 6px;
     overflow: hidden;
-    box-shadow: 1px 25px 0 -22px @focus-glow, 0px 25px 6px -19px @focus-glow;
+    box-shadow: 0px 19px 0 -16px @focus-glow, 1px 18px 6px -12px @focus-glow;
```

Comment:
use the box-shadow [mixin](https://github.com/dequelabs/pattern-library/blob/master/lib/variables.less#L137) here and the one below on line 29

Reply:
Did not think our mixin was setup for multiple, but I just figured out this fun little fact. In newer versions of less you can do `.box-shadow(0px 19px 0 -16px @focus-glow, 1px 18px 6px -12px @focus-glow;);` that extra semicolon is important. Will pick up on the comma separate as separate box-shadows and not args. Didn't know that. 👍